src/scripts/tests/test_update_ticket.py

```
@@ -0,0 +1,218 @@
+depends_on: []
+covers: [CAP-4]
+risk: 3
+created: 2026-08-01
+---
+
+# ALRT-3 — Alert rules
+"""
+
+
+def run(*args):
+    proc = subprocess.run([sys.executable, str(SCRIPT), *args],
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)